### DISCUSSION TITLE
Allow SN comment thread embedding?

### DISCUSSION THREAD

**Discussion by huumn** [stackernews/stacker.news#3004] [Ideas]
I was recently asked by Car at https://thrillerx.com if it were possible to do this on his articles. I'm uncertain of how much work is involved, but it'd be interesting to do if it's not too hard.

This book was written by some disqus engineers and might be helpful (if outdated):
https://www.amazon.com/Third-Party-JavaScript-Ben-Vinegar/dp/1617290548

There might be superior resources now ...

**Comment by dillon-co** [comment]
Still interested in doing this @huumn 

I got a lot of experience with embeddings in sql databases last year and can definitely knock this one out. 

To clarify, do you mean Vector embeddings?

**Comment by huumn** [comment]
No this is about embedding comment threads on other websites
